nucleus/payara-modules/nucleus-microprofile/config-service/src/main/java/fish/payara/nucleus/microprofile/config/spi/ConfigProviderResolverImpl.java

```diff
@@ -426,9 +428,9 @@ private Properties getPropertiesFromFile(ClassLoader appClassLoader, String file
             try (InputStream is = url.openStream()) {
                 p.load(url.openStream());
```

Comment:
Is this call twice to openStream intentional or should this variable _is_ be used for the second call?

Reply:
Clearly I was so focused on fixing my problem I didn't review the code for additional issues!